### DISCUSSION TITLE
Fatal error:  Cannot redeclare strtoloweru() after update

### DISCUSSION THREAD

**Discussion by glow12121** [datenstrom/community#856] [Report a bug] [answered]
Hi,

I am having this error with my Yellow instance. I recently updated the program, not sure if it is related.

```
[user@pc yellow]$ php yellow.php build
PHP Fatal error:  Cannot redeclare strtoloweru() (previously declared in /home/user/yellow/system/extensions/core.php:3702) in /home/user/yellow/system/extensions/core.sync-conflict-20230411-232551-3VJV4WB.php on line 3701
[user@pc yellow]$ php yellow.php update all
PHP Fatal error:  Cannot redeclare strtoloweru() (previously declared in /home/user/yellow/system/extensions/core.php:3702) in /home/user/yellow/system/extensions/core.sync-conflict-20230411-232551-3VJV4WB.php on line 3701
```

**Comment by annaesvensson** [comment]
What does the log file `system/extensions/yellow-website.log` say? Which file syncing software do you use?

**Reply by glow12121** [reply]
Thanks for the answer!

These are this year's log from the file you mentioned:

```
2023-03-19 12:19:46 info Update extension 'Update 0.8.92'
2023-03-19 12:19:52 info Apply patches for Datenstrom Yellow 0.8.21
2023-04-06 00:50:38 info Update extension 'Core 0.8.106'
2023-04-06 00:50:43 info Update Datenstrom Yellow 0.8.22, PHP 8.2.4, Built-in 8.2.4, Linux
2023-04-11 13:47:51 info Update extension 'Core 0.8.107'
```

I normally use Megasync but because it has been crashing recently on one of my computers, I have used Syncthing.

**Reply by annaesvensson** [reply]
Thank you. It's always good to have a log file in bug reports, strange, I expected there to be errors too.

**Comment by schulle4u** [comment, marked as answer]
I think this is a file syncing conflict. Please delete all files those file names contain `sync-conflict`.

**Reply by glow12121** [reply]
Yes, this has solved the issue. Thanks, @schulle4u!
